### DISCUSSION TITLE
How can I add nativeCrs when add service to ldproxy

### DISCUSSION THREAD

**Discussion by wfsiu** [ldproxy/ldproxy#990] [Configuration]
Hi, I'm writing POST request for ldproxy to add service. I need to handle Hong Kong 1980 Grid System (EPSG:2326) (ref: https://epsg.io/2326), so I need to change nativeCrs to 2326. Below example to add service:

domain:
`https://ldproxy.local:7080/rest/admin/services/?f=json`

POST json:
```
{
"featureProviderType":"WFS",
"url":"https://arcgis.server.domain/server/services/test_dataset/MapServer/WFSServer?service=wfs&request=GetCapabilities",
"id":"test_id",
"serviceType":"OGC_API",
"providerType":"FEATURE",
"label":"test_label",
"nativeCrs":{"code":"2326","forceAxisOrder":"NONE"}
 }
```

However, ldproxy always return error:
```
Server Error with ID e1b1474ef7b87776: Cannot deserialize value of type `java.lang.String` from Object value (token `JsonToken.START_OBJECT`)
 at [Source: (org.glassfish.jersey.message.internal.ReaderInterceptorExecutor$UnCloseableInputStream); line: 8, column: 14] (through reference chain: java.util.LinkedHashMap["nativeCrs"])
```

This should be bug in EpgsCrs to handle string to Force enum. Could you please have a check and fix it? Thanks.

**Comment by azahnen** [comment]
Beware that this is an undocumented API and therefore not part of the product, it may change or be removed completely at any time.

That being said, I think your use case should work. `code` is an integer, so this should do the trick: 

```
"nativeCrs":{"code":2326,"forceAxisOrder":"NONE"}
```

**Comment by wfsiu** [comment]
> Beware that this is an undocumented API and therefore not part of the product, it may change or be removed completely at any time.
> 
> That being said, I think your use case should work. `code` is an integer, so this should do the trick:
> 
> ```
> "nativeCrs":{"code":2326,"forceAxisOrder":"NONE"}
> ```
Do you mean my add service api are undocumented or just nativeCrs part is undocumented?

**Comment by azahnen** [comment]
The API is undocumented.
